README.md

```diff
@@ -36,11 +36,11 @@
 
 ## Data/Telemetry
 
-Windows distributions of this project may collect usage data and send it to Microsoft to help improve our products and services. See the [privacy statement](docs/Privacy.md) for more details.
+Windows distributions of this project may collect usage data and send it to Microsoft to help improve our products and services. See the [privacy statement](https://github.com/microsoft/onnxruntime/blob/main/docs/Privacy.md) for more details.
```

Comment:
But what if this is not in the main branch? Using an abs link will link to a wrong version of document.